config/settings/base.py

```diff
@@ -281,3 +280,5 @@
 
 # Your stuff...
 # ------------------------------------------------------------------------------
+FONDY_MERCHANT_ID = env.int("FONDY_MERCHANT_ID", default=1397120)
+FONDY_SECRET_KEY = env.str("FONDY_SECRET_KEY", default="test")
```

Comment:
Is this safe env vars defaults?

Reply:
Yes, these defaults are safe. It's only testing data, published on fondy api docs

Reply:
Just saw that there is comment above that looks... pretty weird...